nevergrad/benchmark/experiments.py

```diff
@@ -891,6 +892,30 @@ def rocket(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
+@registry.register
+def control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """MuJoCo testbed. Learn linear policy for different control problems.
+    Budget 500, 1000, 3000, 5000."""
+    seedg = create_seed_generator(seed)
+    funcs = [control.Ant(num_rollouts=5, random_state=seed),
+             control.Swimmer(num_rollouts=5, random_state=seed),
+             control.HalfCheetah(num_rollouts=5, random_state=seed),
+             control.Hopper(num_rollouts=5, random_state=seed),
+             control.Walker2d(num_rollouts=5, random_state=seed),
+             control.Humanoid(num_rollouts=5, random_state=seed)
+             ]
+    optims = ["RandomSearch", "Shiwa", "CMA", "PSO", "OnePlusOne"]
```

Comment:
I would add "NGOpt2", "DE", "Zero", "Powell", "Cobyla", "LaMCTS", "MetaTuneRecentering" and run tests
@LaurentMnr95, if you have faith for this :-)  (see https://facebookresearch.github.io/nevergrad/benchmarking.html)

(We might change the instrumentation so that the algorithms can use the structure of the controller, but this requires a bit more of work, I've done it for other benchmarks)

Reply:
"NGOpt2" and ""LaMCTS" are not registered ? (even after rebase from master).

Reply:
Yes it looks like it is not registered.

Reply:
Should we wait for these algorithms to be available ? So that we can reproduce results from LaMCTS paper before merging.

Reply:
I merged with <<lamcts>> branch, which contains "Lamtcs" optimizer. But got issues configuring MuJoCo and Gym

Reply:
For mujoco installation, I followed steps from https://github.com/openai/mujoco-py

Reply:
do you have a license for mujoco?

Reply:
Yes, I have license. It is free for student.

Reply:
Here are the last plots with all optimizers: 
![fight_all](https://user-images.githubusercontent.com/25831515/92574773-d407a900-f287-11ea-9fe5-5de1eaaf92cf.png)

Reply:
is Lamcts already available in master ?

Reply:
I merged it from Lamcts branch.

Reply:
You can merge your current branch with the lamcts one.

Reply:
ok thanks :)